### PR TITLE
Automated release of 21.7.0 upstream release 21.0 1639590783

### PR DESCRIPTION
**Pull request by yliaog**
#### What type of PR is this?
/kind bug

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
+# v21.7.0
+
+Kubernetes API Version: v1.21.7
+
+### Uncategorized
+
+
+# v21.0.0-snapshot
+
+Kubernetes API Version: To Be Updated
```
Comment (roycaihw) [inline CHANGELOG.md:10]: Could you remove L5-10?

Comment (yliaog) [reply CHANGELOG.md]: done

**Pull request merged**